backend/app/rest/log_records_routes.py

```diff
@@ -34,22 +34,26 @@ def get_log_records():
     Get RESULTS_PER_PAGE log records. Will return the log records of corresponding to the page you're on. Can optionally add filters.
     """
     page_number = 1
+    return_all = False
     try:
         page_number = int(request.args.get("page_number"))
     except:
```

Comment:
Want to verify here the point of the nested try/except is because if we're given a page number, we don't care about return all at all? Basically `page_number` takes precedence over `return_all`

Reply:
yeah if there's no `page_number`, `return_all` takes precedence. If there's no `return_all`, the page_number will default to 1